tests/test_actions.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
```

Comment:
1. Add docstrings to the test methods explaining what's being tested.
2. Add assertions to actually validate the CSV. Knowing the method was called correctly is not as useful as knowing the method's output is valid. (Too much mocking can be a bad thing.)